experimental/examples/bubblebench/editable-shared-tree/src/client.ts

```diff
@@ -3,14 +3,12 @@
  * Licensed under the MIT License.
  */
 import { IClient } from "@fluid-example/bubblebench-common";
-import { brand, EditableField, FieldKey } from "@fluid-internal/tree";
-import { Bubble } from "./bubble";
-import { ClientTreeProxy, BubbleTreeProxy } from "./schema";
+import { EditableField } from "@fluid-internal/tree";
+import { BubbleWrapper } from "./bubble";
+import { Client, FlexBubble } from "./schema";
 
-export class Client implements IClient {
-	static bubblesFieldKey: FieldKey = brand("bubbles");
-
-	constructor(public readonly clientTreeProxy: ClientTreeProxy) {}
+export class ClientWrapper implements IClient {
```

Comment:
Also curious if we can elide this wrapper.

(The other bubblebench implementations manipulate the 'Client.bubbles[]' array directly from AppState rather than forward the call to 'AppState.increase/decreaseBubbles()' to the IClient.)

Reply:
See https://github.com/microsoft/FluidFramework/pull/14243#discussion_r1132971797